dbservice/boltdb/checker.go

```diff
@@ -0,0 +1,84 @@
+package boltdb
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/aquasecurity/postee/v2/dbservice/dbparam"
+	"github.com/aquasecurity/postee/v2/log"
+	bolt "go.etcd.io/bbolt"
+)
+
+func (boltDb *BoltDb) CheckSizeLimit() {
+	if dbparam.DbSizeLimit == 0 {
+		return
+	}
+	boltDb.mu.Lock()
+	defer boltDb.mu.Unlock()
+
+	db := boltDb.db
+
+	if err := db.Update(func(tx *bolt.Tx) error {
+		b := tx.Bucket([]byte(dbparam.DbBucketName))
+		if b == nil {
+			return nil
+		}
+		c := b.Cursor()
+		size := 0
+		for k, v := c.First(); k != nil; k, v = c.Next() {
+			size += len(v)
```

Comment:
we get the size in bytes, but [readme](https://github.com/aquasecurity/postee/blob/dev/README.md#settings) says Mb.